Added a Blueberry Muffin / Cake recipe (#509)
LukeSmithxyz/based.cooking: src/blueberry-muffins.md

# Blueberry Muffins

![blueberry-muffins](/data/pix/blueberry-muffins.webp)

- ⏲️ Prep time: 20 min
- 🍳 Baking time: 20 min
- 🍽️ Servings: 12

## Equipment

-At least 12 muffin tins / A cake tin and optionally a hand mixer

## Ingredients

- 250g / 9 oz Blueberries 
- 125g / 4½ oz Butter*
- 150g / 5½ oz Sugar (Of course  you can lower the amount to your liking)
- 1 Vanilla Pod or Teaspoon of vanilla sugar
- 2 Eggs
- 125ml / 4.22 fluid ounces of Milk
- 300g /  11 oz Flour
- 2 Teaspoons of Backing Powder
- 125g / 4½ oz Powdered sugar (optional)

> - Half a lemon (optional)
> *If you don't have Butter, you can substitute it with 100g of Oil / 3½ oz of Oil suited for baking. Notice the measurement in grams / ounces and not milliliter and fluid ounces

Reviewer: ```suggestion
- Half a lemon (optional)

*If you don't have Butter, you can substitute it with 100g of Oil / 3½ oz of Oil suited for baking. Notice the measurement in grams / ounces and not milliliter and fluid ounces
```

Adding an extra space. Otherwise markdown will consider the `*` line t be appended to the `half a lemon` line

## Directions

> 1. If using a vanilla pod, wash it with water, extract it's vanilla and add it to the molten butter. Put the now empty vanilla pod into the milk. This way every single bit of the vanilla get's extracted

Reviewer: ```suggestion
1. If using a vanilla pod, wash it with water, extract it's vanilla and add it to the molten butter. Put the now empty vanilla pod into the milk. This way every single bit of the vanilla gets extracted
```

2. Wash the blueberries

> 2. For the liquid ingredients, melt the butter, do not cook it

Reviewer: You have 2 `2.`s in a row. I don't care if you fix it, but thought I'd point it out in case you cared. Markdown fixes numbering for you anyways so whether or not the numbers are correct makes no difference in the end.

Reviewer 2: Thank you for pointing that out! I'll fix it!
And thank you so much for all the other suggestions and looking through the file in general.
I'll make sure to step up my english-game for future recipes

Reviewer: np. I assume your portfolio page is also in development? And just so you know, the 2 links to your portfolio on your website don't point to where you think they do.

one of them points to `https://alexplatonov.com/alexplatonov.com/portfolio` and the other points to `https://alexplatonov.com/halexplatonov.com/portfolio`.

The 2 anchor tags you have are these:

```html
<a href="alexplatonov.com/portfolio">
<a href="halexplatonov.com/portfolio">
```

If you don't have a protocol specified, HTML assumes these are relative paths on your site, not URLs

You can change them to any one of these and they will work:

```html
<a href="portfolio">
<a href="/portfolio">
<a href="https://alexplatonov.com/portfolio">
```

assuming 'portfolio' ends up being a real route to something.

Reviewer 2: Okay, the numbering is now fixed, thank's for pointing that out!
Oh! You are right! I have not even noticed that i put in broken links. I now gave them a proper linking that indicate a work in progress! :)
It's always good to have another pair of eyes watching over the code, thank you!

3. Add the extracted vanilla / vanilla sugar to the molten butter
4. Add the milk to the molten butter. Take the vanilla pod out of the milk if you have used one before adding it
5. Add the two eggs to the molten butter, no need to separate yolk from the egg white
6. Add the sugar to the molten butter
7. Mix the aforementioned ingredients up

> 8. For the dry ingredients, add the backing powder and salt to the flour

Reviewer: ```suggestion
8. For the dry ingredients, add the baking powder and salt to the flour
```

9. Mix both the liquid ingredients with the dry ingredients, the dough should be thick enough to not be completely fluid but also be just fluid enough to be easily distributed across the muffin tins
10. The blueberries should have lost most of the water from washing by now, carefully mix them into the dough

> 11. Distribute the dough across the muffin tins and put them into the preheated oven (190 °C / 375 Fahrenheit), the muffins should be ready after 20 minutes

Reviewer: ```suggestion
11. Distribute the dough across the muffin tins and put them into the preheated oven (190°C / 375°F), the muffins should be ready after 20 minutes
```

12. Extract the juice of the lemon and slowly add it to the powdered sugar. Mix it with a whisk until you get a thick mass (optional)
13. Distribute the coating across the top of the muffins and let them cool (optional)

## Cool Tip:

> Muffins are great for sharing at gatherings, but you don't necessary have to waste time and resources on muffin tins. You can always just pour the dough into a cake tin! However, keep in mind that the baking temperature and time should be adjusted. For me 150 °C / 300 Fahrenheit at 50mins. worked the best. 

Reviewer: ```suggestion
Muffins are great for sharing at gatherings, but you don't necessary have to waste time and resources on muffin tins. You can always just pour the dough into a cake tin! However, keep in mind that the baking temperature and time should be adjusted. For me 150°C / 300°F for 50 mins. worked the best. 
```

![blueberry-muffins](/data/pix/blueberry-muffins-01.webp)

## Contribution

> - Alexander Platonov - [website](https://alexplatonov.com), [donate](https://alexplatonov/donate.com)

Reviewer: ```suggestion
- Alexander Platonov - [website](https://alexplatonov.com), [donate](https://alexplatonov.com/donate)
```

I assume the donate URL should be this suggestion, but that page doesn't exist so I don't know.

Tried finding your donate page on your website but didn't see it either, and it looks like some of the links are broken, just fyi

Reviewer 2: Thank you for testing out the links!
The Donate page is currently developed by me and should be online later this day, i just wanted to have the link already in the .md file

- Pictures taken by Alexander Platonov

> ;tags: baking baked sweet cake dessert

Reviewer: ```suggestion
;tags: baking sweet cake dessert
```

I don't really know why we having a `baking` and `baked` tag, but the `baked` tag doesn't actually have any 'baking' on it, so may as well remove it from this recipe.

Reviewer 2: Yea you are right, i've read in the initial readme that i should use already existing ones and thought that "baked" was one of those. But it's probably better to have lesser tags, i'll keep that in mind!

# Blueberry Muffins

> ![blueberry-muffins](/data/pix/blueberry-muffins.webp)

Reviewer: ```suggestion
![blueberry-muffins](pix/blueberry-muffins.webp)
```

The structure of the website is a little different than the repo, so the correct path for the images is without `/data/`.

Reviewer 2: I see, makes sense!
